docs/articles/architecture/arch-vertical-configurations.rst

```diff
@@ -10,11 +10,110 @@ Each vertical configuration describes a sub-system of the product.
 Update management
 =================
 
-The software update components are responsible for platform level software update.
+The software update components are responsible for platform level software
+update. This sub-system is critical in an automotive context since it allows to
+deploy new functionalities, fix important bugs and security issues or update
+assets such as driving data. A well designed update sub-system can save a lot
+of money by avoiding products recalls.
```

Comment:
I think we can add that not having to go to an auto repair shop to do this is a neat convenience plus probably a money saver.